Unrecoverable semantic error(s) found.
m4.module:1:
  Semantic Error: Bad import cycle found: m4, own m1, m2, own m4
